nixos/modules/services/networking/rosenpass.nix

```diff
@@ -0,0 +1,220 @@
+{ config
+, lib
+, options
+, pkgs
+, ...
+}:
+with builtins;
+with lib; let
+  cfg = config.services.rosenpass;
+  opt = options.services.rosenpass;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.services.rosenpass = with types; {
+    enable = mkEnableOption "Rosenpass";
```

Comment:
```suggestion
    enable = mkEnableOption (lib.mdDoc "Rosenpass");
```

Reply:
I thought that `lib.mdDoc` should not be used anymore (see other comment in this PR)?!

Reply:
Last thing I heard from the docs people was to still use it to make backports easy.

Reply:
@fricklerhandwerk, @mightyiam could you please tell me what the current policy on `lib.mdDoc` is? If it should still be used to make backporting easier, then the comment above its definition is very misleading:
https://github.com/NixOS/nixpkgs/blob/8ab2b1221df4ffaafe779f7c2e4880977695d8d5/lib/options.nix#L346-L349

Reply:
@Pennae is the authority on this

Reply:
Alright. Added.